aleth/main.cpp

```diff
@@ -394,12 +393,13 @@ int main(int argc, char** argv)
 
     po::options_description generalOptions("GENERAL OPTIONS", c_lineWidth);
     auto addGeneralOption = generalOptions.add_options();
-    addGeneralOption("db-path,d", po::value<string>()->value_name("<path>"),
+    addGeneralOption("data-dir,d", po::value<string>()->value_name("<path>"),
         ("Load database from path (default: " + getDataDir().string() + ")").c_str());
```

Comment:
Option description should be changed (Maybe `Load config files and keystore from path`)

Reply:
Ah yes, thank you for catching that!